LoopTests/Models/RemoteActionTests.swift

```diff
@@ -0,0 +1,108 @@
+//
```

Comment:
I needed to update the RemoteCommandTests with these and change a few that are unnecessary.

I added more tests to LoopKit, which test the individual remote action types themselves which should cover all the cases I know of.